### ISSUE TITLE
Login failure with special characters in password

### ISSUE THREAD

**Issue by Matze177** [robinrodricks/FluentFTP#1652]
**FTP Server OS:** Windows

**FTP Server Type:** Windows Server IIS

**Client Computer OS:** Windows

**FluentFTP Version:** 51.1.0

**Framework:** .NET 4.8

We were trying to login to the FTP server using FTPS and failing with the server response "530 User cannot login.". Login using a different FTP-Client (FileZilla) worked.

Using the logger and configuration parameters, we finally noticed that the the emitted password differed from the provided value. The character ">" got prefixed by a "^"-character that was not in original value. Changing the password by removing the character fixed the problem.

**Logs :**
The problem occured near production and was fixed by the workaround, so: sorry, no logs available.

**Comment by FanDjango** [comment]
Ok, after some research, here is what is going on.

> The character ">" got prefixed by a "^"-character

Encoding problem.

**In your code**, that instantiates the FluentFTP client, do you provide the password as a string in UTF-8 encoding? In the further processing within FluentFTP, this password string is emitted in ASCII encoding, so special characters will "seem" to be prefixed by a code page byte.

Note that an immediate string representation such as `@"password"` will be encoded according to the settings of the IDE (Visual Studio?) that you are using.

FileZilla works because you type the password into the input field of its GUI, and **FileZilla's code** uses the correct encoding (i.e. ASCII).

By the way, later on after a successful logon, if the server advertises UTF-8 capability, you **may** use special characters (such as within file names) and provide them to the FluentFTP API in UTF-8 encoding.

**Comment by Matze177** [comment]
Hey there,

with further testing based on your hints, I can confirm that it's an IDE-issue, although it's not encoding-related. It's a rather odd problem of Visual Studio handling debug parameters passed in quotes: 

The password was modified before passing it to our application, so the magic character is already present before the FluentFTP-component is even instanciated.

Without the quotes, this actually works ... I found several (open) tickets in the VS-forums that mention the problem in different variations.

Note that we used FileZilla only to verify that the credentials are generally working.

**No issue at FluentFTP here**
